### PR TITLE
EMP-676 Create tests for update printer procedure

### PR DESCRIPTION
**Pull request by DanielUltimaker**
Created tests for:
- Update from the stable channel
- Update from the USB drive
- Update to the same version
- Reboot several times (regression test for an old issue)

### CODE REVIEW

File: tests/test_system/FirmwareInstallation_test.py [hunk 1]
```diff
@@ -31,16 +31,24 @@ def setUp(self):
             raise FileNotFoundError
 
 
-    @pytest.mark.skip(reason="pending to be reviewed")
-    def test_01_update_specific_version_from_USB(self):
+    def test_01_update_from_USB_specific_version(self):
+        old_version = ApiMethods.get_printer_version(_PRINTER.printerIp)
```
[the remark this answers is not in the document]
Comment (DanielUltimaker) [reply tests/test_system/FirmwareInstallation_test.py]: True, I have modified the method to return a text directly.

File: tests/test_system/FirmwareInstallation_test.py [hunk 2]
```diff
@@ -49,31 +57,35 @@ def test_02_update_new_version_from_stable(self):
...
         current_version = ApiMethods.get_printer_version(_PRINTER.printerIp)
-        if current_version.text != _VERSION_TO_UPDATE:
-            PrinterActions.update_printer(_VERSION_TO_UPDATE, "stable", _PRINTER.printerIp)
-            res = PrinterActions.check_printer_update_successfully(_PRINTER.printerIp)
-            assert res is True, "The printer has not triggered the successful update message."
-        PrinterActions.update_printer(_VERSION_TO_UPDATE, "stable", _PRINTER.printerIp)
-        res = PrinterActions.check_printer_update_successfully(_PRINTER.printerIp)
-        assert res is False, "The printer has triggered the successful update message and should not.."
 
+        for i in range(2):
```
[the remark this answers is not in the document]
Comment (DanielUltimaker) [reply tests/test_system/FirmwareInstallation_test.py]: Right now, I am not able to see the rationale behind this loop, so I have removed it!

[the remark this answers is not in the document]
Comment (jackha) [reply tests/test_system/FirmwareInstallation_test.py]: You've removed it after discussing it I assume @DanielUltimaker ;-)

[the remark this answers is not in the document]
Comment (DanielUltimaker) [reply tests/test_system/FirmwareInstallation_test.py]: Yes, of course!

**Pull request merged**